types/jira-client/index.d.ts

```diff
@@ -6,8 +6,6 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-/// <reference types="node" />
```

Comment:
This file depends on `@types/node/fs.d.ts`, which AFAIK, needs `<reference types="node" />` for [`@definitelytyped/publisher`][dt-publisher] to correctly infer the dependencies when generating the `package.json` file.

[dt-publisher]: https://github.com/microsoft/DefinitelyTyped-tools/tree/master/packages/publisher